bokehjs/src/coffee/common/plot_widget.coffee

```diff
@@ -15,7 +15,7 @@ class PlotWidget extends ContinuumView
 
   bind_bokeh_events: () ->
 
-  request_render: () ->
+  request_render: () =>
```

Comment:
I agree. By setting the img.src inside the closure (instead of calling request_render) made this unnecessary anyway.